### ISSUE TITLE
Drop default log prettifier

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/directus#16060]
Directus currently ships with pino-pretty (formerly pino-colada) enabled by default, with a environment variable to disable that and go back to "raw" mode. 

The default logger uses the programmatic logger transport, which isn't the best for performance. Instead, the prettifier should be ran on the CLI: `directus start | pino-pretty`. 

We should remove the ootb default included log prettifier in favor of relying on external transports.

**Comment by br41nslug** [comment]
How will this external transport approach translate to Docker setups? Or will they still default to pino-pretty baked in?

**Comment by rijkvanzanten** [comment]
@br41nslug Really depends on how they're doing log ingestion. You can run it with like `docker run directus/directus | pino-pretty`, or you can install a log transport into your image, and then update the CMD statement. There's ways!

**Comment by rijkvanzanten** [comment]
Linear: [ENG-161](https://linear.app/directus/issue/ENG-161/drop-default-log-prettifier)

**Comment by molszanski** [comment]
The APM / DD  integration would be much appreciated

**Comment by rijkvanzanten** [comment]
> The APM / DD integration would be much appreciated

Set Directus' `LOG_STYLE` to `raw`, and you should be able to use https://www.npmjs.com/package/pino-datadog

**Comment by molszanski** [comment]
@rijkvanzanten , yeah, I've been using bot `LOG_STYLE`  ([directus docs: Configuration Options / General](https://docs.directus.io/self-hosted/config-options.html#general)) and pino-datadog in a pipe / CLI mode. 
There is still some configuration to go there :)

**Comment by rijkvanzanten** [comment]
"Polishing the setup" isn't really something I can _do_ without more information 🙂 Mind opening [a new feature request](https://github.com/directus/directus/discussions/new?category=feature-requests) detailing what you're imagining? Thanks!

**Comment by molszanski** [comment]
Oh I am sorry, I will be more precise next time. 

My intention was to say that I plan to work on updating our current production DD / directus Pino setup. Will share the results, Setup, tricks, etc :)

**Comment by rijkvanzanten** [comment]
The prettifier we currently use shows more useful information for errors, so this is no longer relevant.
